addons/sys_prc117f/menus/fnc_changeMenu.sqf

```diff
@@ -22,11 +22,11 @@ params["_newMenu"];
 
 
 // clear menu data
-SET_STATE("menuSelection" , 0);
+SET_STATE(menuSelection , 0);
```

Comment:
Due to the way the macros works this becomes some thing like ` ["menuSelection "`